packages/w3c-webdriver/test/13-element-state.test.ts

```diff
@@ -42,6 +41,26 @@ describe('Element State', () => {
     });
   });
 
+  describe('getCssValue method', () => {
```

Comment:
I have not changed this. Just changed the order to be the same as in specification